src/routes/transactions/mappers/queued-items/queued-items.mapper.ts

```diff
@@ -132,4 +135,27 @@ export class QueuedItemsMapper {
         },
     );
   }
+
+  /**
+   * Adjusts the timestamps of transactions array by given offset
+   * @param transactions transactions to offset the timestamp of
+   * @param timezoneOffset UTC timezone offset in milliseconds
+   */
+  private getTimezoneOffsetTransactions(
+    transactions: MultisigTransaction[],
+    timezoneOffset: number,
+  ): MultisigTransaction[] {
+    if (timezoneOffset === 0) {
+      return transactions;
+    }
+
+    // We clone so as to not modify the original dates
+    return structuredClone(transactions).map((transaction) => {
+      // No need to set the `executionDate` as it will not exist in the queue
+      transaction.modified?.setMilliseconds(timezoneOffset);
```

Comment:
Maybe I'm missing something here but why not use `setUTCMilliseconds` here also?

```suggestion
      transaction.modified?.setUTCMilliseconds(timezoneOffset);
```